sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -228,6 +228,7 @@ export interface HealthcareAssertion {
 export interface HealthcareEntity extends Entity {
     assertion?: HealthcareAssertion;
     dataSources: EntityDataSource[];
+    normalizedText?: string;
```

Comment:
curious, in which cases will the normalized text not be returned?

Reply:
not sure tbh

Reply:
could u follow up with the service team and understand more why it has to be nullable?

Reply:
Replied in private.